CryptoTechReminderSystem.Test/RemindUserTests.cs

```diff
@@ -9,7 +9,7 @@ namespace CryptoTechReminderSystem.Test
 {
     public class RemindUserTests
     {
-        class MessageSenderSpy : IMessageSender
+        class SlackGatewaySpy : ISlackGateway
```

Comment:
Do you need to specify `Slack` in the interface name? 

IMO your previous name was slightly better at revealing the intent of the interface, and allows you to have different implementations of it. For example, if it was called `IMessageSendingGateway`, you could have `EmailMessageGateway` or `SlackMessageGateway` in the future which implement this with different delivery mechanisms

Reply:
We've renamed `ISlackGateway` to `IMessageSender` in a6d32b85d4dc88b70afac8e0f4718caafe2da5e4. Thank you!